resources/metric.mjs

```diff
@@ -10,9 +10,13 @@ export class Metric {
             throw new Error(`Invalid metric.name=${name}, expected string.`);
         this.name = name;
         this.unit = unit;
+        this.description = "";
 
         this.mean = 0.0;
-        this.geomean = 0.0;
+        Object.defineProperty(this, "geomean", {
+            writable: true,
+            value: 0,
+        });
```

Comment:
Is there a reason you're using defineProperty here? If yes can you please add a comment explaining why?

Reply:
This is the `Don't serialize "geomean" in the JSON data` part, since it's non-enumerable.

Reply:
The comment below was mentioning circular references though, it's not the case here I guess.
You only want to avoid that it's serialized? The reason isn't clear to me, that's why explaining the "why" in a comment would be useful.

Reply:
added more explicit comment.